tools/clang/test/CodeGenSPIRV/binary-op.arithmetic.scalar.hlsl

```diff
@@ -0,0 +1,121 @@
+// Run: %dxc -T ps_6_0 -E main
+
+void main() {
+    int a, b, c;
+    uint i, j, k;
+    float o, p, q;
+
+// CHECK:      [[a0:%\d+]] = OpLoad %int %a
```

Comment:
These tests are so sweet sweet sweet.  SWEET SWEET SWEET.

I love the fact that we've put in enough pieces together to make this test really easy to write.

(I'm going to scratch my back for two key pieces: the friendly names for disassembly so that we get nice type names, and effcee / FileCheck.)